src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -94,6 +94,8 @@ FixedwingPositionControl::FixedwingPositionControl() :
 	_parameter_handles.heightrate_ff =			param_find("FW_T_HRATE_FF");
 	_parameter_handles.speedrate_p =			param_find("FW_T_SRATE_P");
 
+	_parameters.vtol_type = -1;
```

Comment:
We should also add a `param_get` for this, otherwise it's never updated with `VT_TYPE`.

Reply:
Thanks, I completely missed that. Good catch.